src/jobs/helm_push.yaml

```diff
@@ -0,0 +1,137 @@
+description: "Set version and publish to chart museum"
+
+executor: aws
+
+parameters:
+  chart_name:
+    type: string
+    description: "name of the helm chart"
+  helm_dir:
+    type: string
+    description: "directory containing the helm chart"
+    default: ".helm"
+  app_version:
+    type: string
+    description: "app version in Chart.yaml"
+    default: ""
+  api_version:
+    type: string
+    description: "api version in Chart.yaml"
+    default: ""
+  chart_version:
+    type: string
+    description: "chart version in Chart.yaml"
+    default: ""
+  image_tag:
+    type: string
+    description: "image tag in values.yaml"
+    default: ""
+  charts_repo_domain_ssm:
+    type: string
+    description: "charts repo domain ssm parameter"
+    default: "/codacy/chartmuseum/DOMAIN"
+  charts_repo_user_ssm:
+    type: string
+    description: "charts repo username ssm parameter"
+    default: "/codacy/chartmuseum/BASIC_AUTH_USER"
+  charts_repo_pass_ssm:
+    type: string
+    description: "charts repo password ssm parameter"
+    default: "/codacy/chartmuseum/BASIC_AUTH_PASS"
+  aws_profile:
+    type: string
+    description: "AWS profile to be used"
+    default: "default"
+  region:
+    type: string
+    description: "aws region for the deployment"
+    default: eu-west-1
+  credentials_file_content:
```

Comment:
this entire section is already in: https://github.com/codacy/codacy-orbs/blob/master/src/jobs/aws_deploy.yml#L17
can  we refactor in a way that stays only once in the codebase?
e.g.: build a configurable command and re-use it around

Reply:
We can have a step that creates credentials file and persist it to the workspace for later commands to use, like we do with .version. If you think that is ok I can take care of it.